api/client/service/update.go

```diff
@@ -30,8 +31,8 @@ func newUpdateCommand(dockerCli *client.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.String("image", "", "Service image tag")
-	flags.StringSlice("command", []string{}, "Service command")
-	flags.StringSlice("arg", []string{}, "Service command args")
+	flags.String("command", "", "Service command")
+	flags.String("args", "", "Service command args")
```

Comment:
I have major concerns for shell interpretation. In fact, it is possible that we removed this exact dependency at one time or another because of this issue.